cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -0,0 +1,287 @@
+name "AWS Reserved Instances Recommendations"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications when AWS RI Recommendations are identified. NOTE: These RI Purchase Recommendations are generated by AWS. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/reserved_instances/recommendations) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "medium"
+category "Cost"
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+parameter "param_days" do 
+  category "RI"
+  label "Look Back Period"
+  default "Last 7 Days"
+  allowed_values "Last 7 Days","Last 30 Days","Last 60 Days"
+  type "string"
+end 
+
+parameter "param_service" do 
+  category "RI"
+  label "Service"
+  default "Elastic Compute Cloud (EC2)"
+  allowed_values "ElastiCache","Elastic Compute Cloud (EC2)","Elasticsearch Service (ES)","Redshift","Relational Database Service (RDS)"
+  type "string"
+end 
+
+parameter "param_service_spec" do 
+  category "RI"
+  label "EC2 Specification"
+  allowed_values "Standard","Convertible"
+  default "Standard"
+  description "Note: this parameter will be ignored if the Service parameter is not \"Elastic Compute Cloud (EC2)\""
+  type "string"
+end 
+
+parameter "param_term" do 
+  category "RI"
+  label "RI Term"
+  allowed_values "Any","1 Year","3 Year"
+  default "Any"
+  type "string"
+end 
+
+parameter "param_payment_option" do
+  category "RI"
+  label "(Optional) Payment Option"
+  default ""
+  allowed_values "","No Upfront","Partial Upfront","All Upfront","Light Utilization","Medium Utilization","Heavy Utilization"
+  type "string"
+  description "Leave blank to include all RI purchase options"
+end 
+
+parameter "param_savings_threshold" do
+  category "RI"
+  label "Net Savings Threshold"
+  description "Specify the minimum estimated monthly savings that should result in a recommendation"
+  default 100
+  type "number"
+end
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+end
+
+auth "aws", type: "aws" do
+  version "4"
+  service "ce"
+  region "us-east-1"
+  access_key cred("AWS_ACCESS_KEY_ID")
+  secret_key cred("AWS_SECRET_ACCESS_KEY")
+end
+
+#pagination "aws_pagination" do
```

Comment:
What's up with the pagination block that's commented out?

Reply:
this pagination method wasn't supported when initially developed. but it is now! thanks for pointing it out, i'll update